internal/raft/message/convert.go

```diff
@@ -0,0 +1,971 @@
+package message
```

Comment:
Please make sure that all commends are formed out of full sentences.

Also, use the initialization syntax, and don't create a zero value and then access each field separately.

Also, only use pointers where neccessary. I don't think you will modify the returned structs, so there's no reason to use a pointer.